yb-voyager/cmd/common.go

```diff
@@ -452,3 +452,34 @@ func nameContainsCapitalLetter(name string) bool {
 	}
 	return false
 }
+
+func checkCutoverCompleted() bool {
+	cutoverFpath := filepath.Join(exportDir, "metainfo", "triggers", "cutover")
+	cutoverSrcFpath := filepath.Join(exportDir, "metainfo", "triggers", "cutover.source")
+	cutoverTgtFpath := filepath.Join(exportDir, "metainfo", "triggers", "cutover.target")
+
+	a := utils.FileOrFolderExists(cutoverFpath)
+	b := utils.FileOrFolderExists(cutoverSrcFpath)
+	c := utils.FileOrFolderExists(cutoverTgtFpath)
+
+	return a && b && c
+}
+
+var withStreamingMode bool
+
+func checkWithStreamingMode() {
+	var err error
+	metaDB, err = NewMetaDB(exportDir)
+	if err != nil {
+		utils.ErrExit("error while connecting meta db: %w\n", err)
+	}
+	migrationStatus, err := GetMigrationStatusRecord()
+	if err != nil {
+		utils.ErrExit("error while fetching migration status record: %w\n", err)
```

Comment:
We should reduce the use of `utils.ErrExit()`. Consider returning proper error from new functions and call `ErrExit()` as close to main() as possible.